swag/blocks/system_blocks.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Union
+
+from swag.id import UserId
+
+if TYPE_CHECKING:
+    from ..blockchain import SwagChain
+
+from attr import attrs, attrib
+import arrow
+
+
+from ..block import Block
+
+from ..errors import (
+    InvalidTimeZone,
+    TimeZoneFieldLocked,
+)
+
+from ..currencies import Swag, Style
+
+
+def assert_timezone(self, attribute, timezone):
```

Comment:
Hm. This one is supposed to be a method actually. It is external because it gets assigned in the attrib decorator for several classes (and therefore should only exists in only one module but time bla bla bla).